typescript/src/main.ts

```diff
@@ -112,14 +112,14 @@ function appendMetalTableRow(metal: metals.Metal, table: HTMLTableElement) {
   resultCell.classList.add("result");
   resultCell.id = "result_" + metal.idSuffix;
 }
-function toggleMetal(bool: boolean, metal: metals.Metal) {
+function toggleMetal(isOff: boolean, metal: metals.Metal) {
```

Comment:
Reading this again with the change, I wonder if `willTurnOff` would be clearer (`isOff` suggests it might already be off). Sorry, I know `isOff` was my suggestion but seeing it in practice it's still not clear. Naming booleans is hard!